WordPress-Theme/ruleset.xml

```diff
@@ -21,9 +21,12 @@
 	<!-- No ByteOrderMark allowed - important to prevent issues with content being sent before headers. -->
 	<rule ref="Generic.Files.ByteOrderMark" />
 
-
 	<!-- Check that the I18N functions are used correctly. -->
 	<!-- This sniff can also check the text domain, provided it's passed to PHPCS. -->
 	<rule ref="WordPress.WP.I18n" />
 
+	<!-- Validate and/or sanitize untrusted data before entering into the database.
```

Comment:
I don't think the sniff checks if the data is sanitized, just that it is escaped.